src/main/java/com/epam/bookstoreservice/model/TokenModel.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.bookstoreservice.model;
+
+import com.epam.bookstoreservice.dto.response.TokenResponseDTO;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.springframework.hateoas.RepresentationModel;
+
+/**
+ * the model for tokenResponseDTO
+ */
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class TokenModel extends RepresentationModel<TokenModel> {
```

Comment:
You could move this model to a new package called 'hateoas/model' since this feature is related to it.